src/lib/plugin.js

```diff
@@ -418,6 +418,7 @@ class FiveBellsLedger extends EventEmitter2 {
   }
 
   * _sendMessage (message) {
+    debug('sending message', message)
```

Comment:
This message and the `converted to ledger message` one should probably use JSON.stringify